source/high-availability.txt

```diff
@@ -105,64 +103,50 @@ unavailable with enough members still available for a primary election.
 Fault tolerance of a four-member replica set is the same as for a three-member replica set because both can withstand a single-node
 outage.
 
+To test if your application can handle a failure of the replica set primary,
+submit a request to :ref:`test primary failover <test-failover>`. 
+When {+service+} simulates a failover event, {+service+} shuts down the current 
+primary and holds an election to elect a new primary.
+
 To learn more about replica set members, see :manual:`Replica Set Members </core/replica-set-members>`. To learn more about replica set elections and 
 voting, see :manual:`Replica Set Elections </core/replica-set-elections>`.
 
 .. _arch-center-distribute-data:
 
 Distribute Data Across At Least Three Data Centers in Different Availability Zones
-````````````````````````````````````````````````````````````````````````````````````
-
-To guarantee that a replica set can elect a primary if a data center
-becomes unavailable, you must distribute nodes across at least three
-data centers, but we recommend that you use five data centers.
-
-If you choose a region for your data centers that supports
-availability zones, you can distribute nodes in data centers in different
-availability zones. This way you can have multiple separate physical data
-centers, each in its own availability zone and in the same region.
-
-This section aims to illustrate the need for a deployment with five data centers.
-To begin, consider deployments with two and three data centers.
-
-Consider the following diagram, which shows data distributed across
-two data centers:
+``````````````````````````````````````````````````````````````````````````````````
+MongoDB provides high availability by storing multiple copies of data across replica sets.
+To ensure that members of the same replica set don't share the same resources, and that 
+a replica set can elect a primary if a data center becomes unavailable, you must distribute 
+nodes across at least three data centers. We recommend that you use at least five data centers.
+
+You can distribute data across multiple data centers by deploying your {+cluster+} to a region that supports multiple availability zones. 
+Each availability zone typically contains one or more discrete data centers, each with redundant power, networking and connectivity, often housed in separate
+facilities. When you deploy a dedicated {+cluster+} to a region that supports availability zones, {+service+} automatically splits your {+cluster+}'s 
+nodes across the availability zones. For example, for a three-node replica set {+cluster+} deployed to a region with three availability zones, 
+{+service+} deploys one node in each zone. This ensures that members of the same replica set are not sharing resources, so that your database remains 
```

Comment:
```suggestion
{+service+} deploys one node in each zone. This ensures that members of the same replica set aren't sharing resources, so that your database remains 
```